phylogenetic/rules/prepare_sequences.smk

```diff
@@ -108,7 +108,7 @@ rule align:
     """
     input:
         sequences = "results/filtered.fasta",
-        reference = config["reference"],
```

Comment:
> Path names
> 
> Interesting file paths for auxilary default files, e.g.:
> 
>  ~/.nextstrain/pathogens/zika/support-nextstrain-run=ON2XA4DPOJ2C23TFPB2HG5DSMFUW4LLSOVXA====/phylogenetic/defaults/reference.gb
> 
> The ==== initially gave me pause, but seems to work fine! I also like that I can easily find the pipeline files in ~/.nextstrain/../../..hashed folder/:

OH interesting. I was initially confused how you were seeing these hashed folders in your file path, then I realized you are probably using the conda runtime. In the docker runtime, the path is just `/nextstrain/pathogen/phylogenetic/defaults/reference.gb`.